scripts/install2

```diff
@@ -331,6 +331,9 @@ sudo systemctl enable cjdns.service
 NEWHOSTNAME=$(sudo grep -m 1 '"ipv6"' /etc/cjdroute.conf | awk '{ print $2 }' | sed 's/[",]//g' | sed "s/.*:/$MESH_NAME-/g")
 export NEWHOSTNAME
 
+# Install nginx without asking
```

Comment:
There is no need to emphasize `without asking`. The comment should just read `# Install nginx` not what it does not do.

Reply:
Done.